linera-chain/src/manager.rs

```diff
@@ -617,15 +645,21 @@ pub struct ChainManagerInfo {
 impl From<&ChainManager> for ChainManagerInfo {
     fn from(manager: &ChainManager) -> Self {
         let current_round = manager.current_round;
+        let pending = manager
+            .confirmed_vote
+            .as_ref()
+            .map(|vote| vote.lite())
+            .or_else(move || manager.validated_vote.as_ref().map(|vote| vote.lite()));
```

Comment:
Also this - it shouldn't happen that both `confirmed_vote` and `validated_vote` are both set to `Some(_)` at the same time.

Reply:
It looks like you made sure of that everywhere.

Alternatively, we could avoid explicitly setting them to `None` and always prefer the highest-round vote, with the confirmed one taking precedence.